client/tests/integration/permissions.rs

```diff
@@ -381,7 +348,7 @@ mod token_parameters {
         let (_rt, _peer, client) = <PeerBuilder>::new().with_port(10_750).start_with_runtime();
         wait_for_genesis_committed(&[client.clone()], 0);
 
-        register_test_token_definition(&client);
+        // register_test_token_definition(&client);
```

Comment:
Why commented?

Reply:
we don't support registering token definitions through ISI anymore, only through validator migration

Reply:
i understand that, but why to preserve commented code?

Reply:
Yes, I left it commented so that someone who will work in #2581 have some glue on how it worked before